### DISCUSSION TITLE
Chinese input cannot be deleted

### DISCUSSION THREAD

**Discussion by bkx1221** [the-airbender/quill_html_editor#79] [General]
hello Pavan Kumar After Chinese input, the br label will be added, causing the content to fail to be deleted

**Comment by the-airbender** [comment]
Hi @bkx1221 ,
Thank you for reporting the issue.
Could you please open a bug issue with some detailed description and sample code to help me understand it better?

Thanks

**Comment by bkx1221** [comment]
![521685515582_ pic_hd](https://github.com/the-airbender/quill_html_editor/assets/65898776/ca556af3-8ac2-40c3-960f-e221dfb18c61)
![531685515685_ pic_hd](https://github.com/the-airbender/quill_html_editor/assets/65898776/bf94e68c-4702-4ccd-84f5-3b5cc720d5a7)
hello @the-airbender   Thank you for taking time out of your busy schedule to see and reply to me   `<p>毕凯旋<br></p>`    `<br>`  label generate The content cannot be deleted   
onTextChanged    Method by adding the following code to solve my current problem

if (htmlText.indexOf("<br>") != -1) {
      htmlTextActive = htmlText.replaceAll("<br>", "<br><br/>");
      await controller.setText(htmlTextActive);
 }
